Tasks/DotNetCoreInstallerV1/versionUtilities.ts

```diff
@@ -0,0 +1,63 @@
+import * as tl from 'vsts-task-lib/task';
+import * as toolLib from 'vsts-task-tool-lib';
+
+export function versionCompareFunction(versionA: string, versionB: string): number {
```

Comment:
Can we use "semver" package for comparing versions?

Reply:
Currently evaluating that only.